src/drop_context.rs

```diff
@@ -89,11 +88,10 @@ where
 /// Swagger Middleware that wraps a `hyper::service::Service`, and drops any contextual information
 /// on the request. Services will normally want to use `DropContextMakeService`, which will create
 /// a `DropContextService` to handle each connection.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct DropContextService<T, C>
 where
     C: Send + 'static,
-    T: hyper::service::Service<ReqBody = hyper::Body, ResBody = hyper::Body, Error = Error>,
```

Comment:
This is going to make the type errors worse, right?

Reply:
Unclear, I'm not sure to what extent the types on the Service object help (they help a lot on the MakeService object due to the indirection).  I've found the errors fairly palatable when using this new client middleware in a downstream crate.